$OpenBSD: patch-get_flash_videos,v 1.1.1.1 2010/10/19 08:09:16 sthen Exp $

any patches sent upstream should be against the modular source,
see http://github.com/monsieurvideo/get-flash-videos

--- get_flash_videos.orig	Mon Oct 18 19:59:37 2010
+++ get_flash_videos	Mon Oct 18 19:59:40 2010
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #################################################
 #
 # This file was automatically generated by utils/combine-perl.pl
